libs/@local/hash-graph-client/api.ts

```diff
@@ -562,10 +530,10 @@ export interface EntityMetadata {
   recordId: EntityRecordId;
   /**
    *
-   * @type {EntityVersion}
+   * @type {EntityTemporalMetadata}
    * @memberof EntityMetadata
    */
-  version: EntityVersion;
+  version: EntityTemporalMetadata;
```

Comment:
This shouldn't still be `version`

Reply:
Applied in [Rename fields in EntityMetadata](https://github.com/hashintel/hash/pull/1854/commits/22de73165691e32a391ec74bce5b2ac2d0a92def)